src/hypofuzz/bayes.py

```diff
@@ -0,0 +1,31 @@
+import math
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from hypofuzz.hypofuzz import FuzzTarget
+
+
+def behaviors_per_input(target: "FuzzTarget") -> float:
+    # an estimator for the number of behaviors the next input will discover.
+    since = target.provider.since_new_branch
+    return (1 / since) if since > 0 else 1
```

Comment:
Function which takes one argument, which is a specific class?  Smells like a method.  (as does `behaviors_per_second`)

I'd also add a todo note for later that we might want to use a larger lookback window than last-one.  We also want to balance between using this-worker and linearized-history, but all that can come later.

Reply:
it does..but I want to keep all the estimator stuff in one place for now, to make it really clear when we're venturing into bayesian land